src/comps/Notifications.tsx

```diff
@@ -0,0 +1,100 @@
+import {
+  CheckCircleIcon,
+  InformationCircleIcon,
+} from "@heroicons/react/20/solid";
+import { motion } from "framer-motion";
+
+export enum NotificationStatusType {
+  SUCCESS = "SUCCESS",
+  ERROR = "ERROR",
+  WARNING = "WARNING",
+}
+
+export type NotificationEventType = {
+  id: string;
+  title: string;
+  type: NotificationStatusType;
+};
+export type NotificationType = NotificationEventType & {
+  handleExitClick: (eventId: string) => void;
+  index: number;
+};
+
+const COLOR_DATA = {
+  [NotificationStatusType.SUCCESS]: {
+    text: "#048848",
```

Comment:
not a priority but it's better to define the colors in the tailwind config for reusability and to easily change in the future if needed